charts/crossplane-keycloak-realm/values-env-config.yaml

```
@@ -346,4 +346,4 @@ users:
       email: emmanuel@pelo.tech
     roles:
       names:
-        - '{{ printf "%s--opendash-admins" (tpl .Values.realm.name $) }}'
+        - opendash-admins
```

I misspoke earlier in Slack. This pattern (role name prefixed by realm name) will work for the roles that we create, but this chart should support referencing arbitrary existing roles, as well. To do so we need to allow the caller (whoever is installing the chart) to specify the exact role names. Meaning we _do_ need to template the values. Apologies. 

It can be a bit simpler than before, though:

```suggestion
        - "{{ tpl .Values.realm.name $ }}--opendash-admins"
```